### PR TITLE
Rework the send delegate to area action using an options dictionary.

### PR DESCRIPTION
**Pull request by kberg**
This is composed of two commits. Reading each separately will probably make it easier to understand.

### CODE REVIEW

File: src/cards/turmoil/MartianMediaCenter.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class MartianMediaCenter extends Card implements IProjectCard {
             pb.megacredits(2);
           });
         }),
-        description: 'Requires that Mars First is ruling or that you have 2 delegates there.Increase your MC production 2 steps.',
+        description: 'Requires that Mars First is ruling or that you have 2 delegates there. Increase your MC production 2 steps.',
```
Comment (sienmich) [inline src/cards/turmoil/MartianMediaCenter.ts:33]: When changing card description, change all the translations, else they are breaking

Comment (sienmich) [reply src/cards/turmoil/MartianMediaCenter.ts:33]: I think better to leave it as it is and in nee PR change the description and translations if u want

**Review by sienmich — APPROVED** [review]

File: src/cards/turmoil/MartianMediaCenter.ts (hunk 1 above)
Comment (kberg) [reply src/cards/turmoil/MartianMediaCenter.ts:33]: Agreed, thank you. I fixed the translation in this PR.

**Review by sienmich — APPROVED** [review]
Thanks, i thought there were more than just polish one :)

**Pull request merged**